Reports/reportModels.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
```

Comment:
I'm not entirely sure what you mean.  Are you suggesting keeping the object schemas in one file, then creating the mongoose Schema and Model objects in another?

All the examples I've seen define the models like this, and after a brief search of Google and Github, I couldn't find any examples in which the schema was in a separate file from the mongoose Schema and Model definitions.